packages/block-library/src/rich-image/editor.scss

```diff
@@ -85,3 +85,13 @@
 .richimage-toolbar__control {
 	padding: 6px;
 }
+
+.richimage__aspect-title {
```

Comment:
Is there no component for this? I see a similar thing in the settings:

<img width="329" alt="Screenshot 2020-06-06 at 17 05 41" src="https://user-images.githubusercontent.com/4710635/83946213-097ba100-a818-11ea-9800-217d034db0f9.png">

Reply:
That's where it is! I though I had seen it somewhere, but I couldn't remember where and I couldn't find a component for it, so I assumed I had imagined it. Looks like it's still just a div, but at least I can grab the class name from it.